plugin/neovim-fuzzy.vim

```diff
@@ -160,16 +159,19 @@ function! s:fuzzy_open(root) abort
 
   " Get all files, minus the open buffers.
   try
-    let files = s:fuzzy_source.find('.', ignorelist)
+    let files_cmd = s:fuzzy_source.find('.', ignorelist)
+
+    " Put it all together.
+    let result = tempname()
+    call writefile(bufs, result)
+    call system(files_cmd . ' >> ' . result)
```

Comment:
This is the first part of the fix - we were running the command after `lcd -`, which kind of misses the point...